app/addons/documents/index-results/containers/IndexResultsContainer.js

```diff
@@ -56,7 +58,8 @@ const mapStateToProps = ({indexResults}, ownProps) => {
     docType: getDocType(indexResults),
     fetchParams: getFetchParams(indexResults),
     queryOptionsParams: getQueryOptionsParams(indexResults),
-    partitionKey: ownProps.partitionKey
+    partitionKey: ownProps.partitionKey,
```

Comment:
This should be in another PR right?

Reply:
It was added in another PR. It's only highlighted because of the added comma at the end of the line.